examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -470,15 +495,29 @@ func extractNonPointerInterfaces(comments []string) (bool, error) {
 	if len(values) == 0 {
 		return false, nil
 	}
-	result := values[0] == "true"
+	result := values[0] == "true" || values[0] == ""
 	for _, v := range values {
-		if v == "true" != result {
+		if v == "true" != result && values[0] != "" {
 			return false, fmt.Errorf("contradicting %v value %q found to previous value %v", interfacesNonPointerTagName, v, result)
 		}
 	}
 	return result, nil
 }
 
+func extractSkipField(comments []string) (bool, error) {
+	values := types.ExtractCommentTags("+", comments)[skipFieldTagName]
+	if len(values) == 0 {
+		return false, nil
+	}
+	result := values[0] == "true" || values[0] == ""
+	for _, v := range values {
```

Comment:
You mean to drop the `=true` variant? I am fine with that as well as long as it is consistent.

You mean the following?
```
tag => true
tag=true => invalid value
tag=false => false
```